lib/unlocked/ManageBoards.dart

```diff
@@ -0,0 +1,844 @@
+                                                            shape: RoundedRectangleBorder(
+                                                                side: BorderSide(
+                                                                    color:
+                                                                        paua),
+                                                                borderRadius:
+                                                                    BorderRadius
+                                                                        .circular(
+                                                                            20.0)),
+                                                            child: Text(
+                                                              "Close",
+                                                              style: TextStyle(
+                                                                  fontSize:
+                                                                      16.0,
+                                                                  color: paua),
+                                                            ),
+                                                          ),
+                                                        ],
+                                                      ),
+                                                    ),
+                                                  );
+                                                },
+                                              );
+                                            },
+                                          )
+                                        ],
+                                      ),
+                                      // Align(
+                                      //   alignment: Alignment.topLeft,
+                                      //   heightFactor: 4,
+                                      //   child: Icon(Icons.share),
+                                      // ),
+                                      // Align(
+                                      //   alignment: Alignment(0.8, -6),
+                                      //   heightFactor: 0.5,
+                                      //   child: Icon(Icons.share),
+                                      // )
+                                    ],
+                                  ),
+                                ),
+                              );
+                            },
+                          ),
+                        ],
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+            )
+          ],
+        ),
+      ),
+    );
+  }
+
+  void slideSheet1() {
+    showModalBottomSheet(
+      context: context,
+      builder: (context) {
+        return Container(
+          color: Colors.white,
+          height: 350,
+          child: Padding(
```

Comment:
I think you can just set padding for `Container` and modify the height of it instead of wrapping in a `Padding` here and each child of `Column`. But, you don't need to change if you don't want, the current look is good already.